docs/index.html

```diff
@@ -337,6 +336,12 @@ <h5>Browser extensions</h5>
             Yes. KeePassXC supports KeePassHTTP-Connector as a legacy browser integration and a newer extension KeePassXC-Browser.
             You can download KeePassXC-Browser for <a href="https://addons.mozilla.org/en-US/firefox/addon/keepassxc-browser/">Mozilla Firefox</a> and
             <a href="https://chrome.google.com/webstore/detail/keepassxc-browser/oboonakemofpalcgghocfoadofidjkkk">Google Chrome / Chromium / Vivaldi</a>.
+            Firefox ESR (52.x) is supported with version 1.0.1 and newer. With the ESR version the following features are disabled
+            because of the older version of WebExtension API:
```

Comment:
Shorten it to 

"Firefox ESR (52.x) is supported, but the following features are disabled because of WebExtension API limitations:"